docs/openapi.yaml

```diff
@@ -213,6 +213,56 @@ paths:
               example:
                 $ref: ./api/transaction/get-mempool-transactions.example.json
 
+  /extended/v1/tx/multiple:
+    parameters:
+      - name: tx_list
```

Comment:
I still have a question about this parameter. Do we know the max # of characters the API will accept in a GET request before throwing an error? So users have a known (even approximate) limit for how many `tx_id`s they can pass here.

Reply:
Currently, I haven't set any limit to it. 
However, I do think there should be one. Do you have any recommendations for it? 
Or we can use the standard 50 or max of 100 as limit.

Reply:
Maybe @CharlieC3 could chime in here. TBH I'm not sure what the limit could be, AFAIK that's more of a web server configuration and/or limitation.

Reply:
Responded in a DM.

Reply:
@MHassanTariq per discussion with @CharlieC3 , my concern has to do more with the specific environment in which the API runs (which can vary greatly by installation), so we can mark it as resolved.